app/assets/stylesheets/_breadcrumbs.scss

```diff
@@ -0,0 +1,11 @@
+.breadcrumb{
```

Comment:
2 style-y suggestions - can you vertically center the stuff in the navbar and can you add some spacing in between / around the stuff in here?